library/std/src/backtrace.rs

```diff
@@ -14,7 +14,7 @@
 //! Backtraces are attempted to be as accurate as possible, but no guarantees
 //! are provided about the exact accuracy of a backtrace. Instruction pointers,
 //! symbol names, filenames, line numbers, etc, may all be incorrect when
-//! reported. Accuracy is attempted on a best-effort basis, however, and bugs
+//! reported. Accuracy is attempted on a best-effort basis, however, any bugs
 //! are always welcome to indicate areas of improvement!
```

Comment:
> any bugs are always welcome to indicate areas of improvement!

This reads like bugs are welcome, which they of course are not :D
It should probably read `and bug reports are always welcome`